iac-extensions/spring-config/src/main/antlr/PropertiesParser.g4

```diff
@@ -9,8 +9,8 @@ propertiesFile
     ;
 
 row
-    : comment
-    | line
+    : line
+    | comment
```

Comment:
How does this reordering relate to the new condition in lexer? Would it still work without reordering, if the lexer now matches comments only at the beginning of a line?